compiler-rt/test/fuzzer/focus-function.test

```diff
@@ -22,9 +21,9 @@ FOCUS_F0: INFO: 0/1 inputs touch the focus function
 RUN: rm -rf %t-corpus
 RUN: mkdir %t-corpus
 # ABC triggers the focus function, others don't.
-RUN: echo ABC$(for((i=0;i<2048;i++)); do echo -n x; done) > %t-corpus/ABC
-RUN: echo AXY$(for((i=0;i<2048;i++)); do echo -n x; done) > %t-corpus/AXY
-RUN: echo ABX$(for((i=0;i<2048;i++)); do echo -n x; done) > %t-corpus/ABX
+RUN: %python -c 'x_str = "ABC" + "x" * 2048; print(x_str)' > %t-corpus/ABC
```

Comment:
Why the `x_str` and not just

`print("ABC" + "x" * 2048)`?

Reply:
It was initially easier for me to use a temp var; but i've updated the code now to bypass it.